pkg/server/commands/reverseexpand/reverse_expand.go

```diff
@@ -392,13 +400,23 @@ func (c *ReverseExpandQuery) reverseExpandTupleToUserset(
 
 		tk := t.GetKey()
 
-		foundObject := tk.GetObject()
+		conditionMet, err := eval.TupleConditionMet(tk, c.typesystem, req.context.AsMap())
+		if err != nil {
+			return fmt.Errorf("failed to evaluate relationship condition: %v", err)
```

Comment:
there's some duplicate error "wrapping" going on here and elsewhere that `eval.TupleConditionMet()` is being used.

```
failed to evaluate relationship condition: failed to evaluate relationship condition: failed to convert context parameter 'param1': for string: unexpected type value '\"bool\"', expected 'string'
```